internal/featuresupport/feature_support_test.go

```diff
@@ -369,19 +369,19 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 			When("GetFeatureSupportList 4.12 with Platform", func() {
 				It(string(*filters.PlatformType)+" "+swag.StringValue(filters.ExternalPlatformName), func() {
 					list := GetFeatureSupportList("dummy", nil, filters.PlatformType, filters.ExternalPlatformName)
-					Expect(len(list)).To(Equal(28))
+					Expect(len(list)).To(Equal(29))
```

Comment:
I don't like the way this test checks the list based on the number of items. This does not in fact verify that the returned support list is correct. In this case, we merely verify that the size of the list grew by one item.

I know this is a pre existing test but we should call out this weakness.

If we are expecting a specific list to be returned for a specific version, I think we should validate that the list contains all of the items it should.

Reply:
I agree, This test was design to check what are the supported feature for a type of platform but got lost in time :) .
there is a new test which is a bit better [here](https://github.com/openshift/assisted-service/blob/master/internal/operators/manager_test.go#L443-L466)


but I will create a task to refactor this, OSC operator needs to be in the next release